pkg/kf/buildpacks/fake/interfaces.go

```diff
@@ -22,15 +22,15 @@ import "github.com/GoogleCloudPlatform/kf/pkg/kf/buildpacks"
 
 // BuilderFactory is implemented by buildpacks.BuilderFactory.
 type BuilderCreator interface {
-	Create(dir, containerRegistry string) (string, error)
+	buildpacks.BuilderCreator
```

Comment:
does it make sense to push the generate commands up next to the definitions now?

Reply:
I'm not sure TBH. It feels a little like clutter to have them next to production code. It feels like test details leaking.

Reply:
Ha! That's almost the same rationale I'd give for pushing them up; because now we have these interfaces that don't do anything.

Maybe we could use the `doc.go` file in every package as a common place to stick our code generators? That would give our option generators a consistent home too.

Reply:
That's true, we already have the options stuff there... What do you think about doing that in a different CL?

Reply:
SGTM, just opened an issue for it